scripts/nginx/readarr.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# Nginx conf for *Arr
+# Flying sausages 2020
+# Refactored by Bakerboy448 2021
+master=$(_get_master_username)
+app_name="readarr"
+
+if ! READARR_OWNER="$(swizdb get $app_name/owner)"; then
+    READARR_OWNER=$(_get_master_username)
+fi
+user="$READARR_OWNER"
+
+app_port="8787"
+app_sslport="7979"
+app_baseurl="$app_name"
+app_configdir="/home/$user/.config/${app_name^}"
+app_servicefile="${app_name}.service"
+app_branch="nightly"
+
+cat > /etc/nginx/apps/$app_name.conf << ARRNGINX
+location /$app_baseurl {
+    proxy_pass          http://127.0.0.1:$app_port/$app_baseurl;
+    proxy_set_header    Host                \$host;
+    proxy_set_header    X-Forwarded-For     \$proxy_add_x_forwarded_for;
+    proxy_set_header    X-Forwarded-Host    \$host;
+    proxy_set_header    X-Forwarded-Proto   \$scheme;
+    proxy_redirect      off;
+
+    auth_basic              "What's the password?";
+    auth_basic_user_file    /etc/htpasswd.d/htpasswd.${master};
+
+    proxy_http_version  1.1;
+    proxy_set_header    Upgrade     \$http_upgrade;
+    proxy_set_header    Connection  \$http_connection;
+}
+
+# Allow the App API
+location /$app_baseurl/api {
+    auth_request    off;
+    proxy_pass      http://127.0.0.1:$app_port/$app_baseurl/api;
+}
+
+# Allow Content
```

Comment:
Don't need content

Reply:
content is still enabled in the official documentation.


```nginx
location /readarr {
  proxy_pass        http://127.0.0.1:8787/readarr;
  proxy_set_header Host $host;
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header X-Forwarded-Host $host;
  proxy_set_header X-Forwarded-Proto https;
  proxy_redirect off;

  proxy_http_version 1.1;
  proxy_set_header Upgrade $http_upgrade;
  proxy_set_header Connection $http_connection;
}
  location /readarr/api {
  auth_request off;
  proxy_pass       http://127.0.0.1:8787/readarr/api;
}

  location /readarr/Content { 
    auth_request off;
    proxy_pass http://127.0.0.1:8787/readarr/Content;
 }
```

Reply:
official docs updated  to match other \*Arrs

```nginx
location /readarr {
    proxy_pass http://127.0.0.1:8787;
    proxy_set_header Host $host;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Host $host;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_redirect off;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $http_connection;
}
# Allow the API External Access via NGINX
location ~ /readarr/api {
    auth_request off;
    proxy_pass http://127.0.0.1:8787;
}
```